test/utilities.js

```diff
@@ -230,6 +230,8 @@ describe('utilities', function () {
     });
 
     expect(expect('foo').result()).to.equal('result');
+
+    delete chai.Assertion.prototype.result;
```

Comment:
It's a great idea to add this so the test cleans up after itself, but we need it to run the cleanup operation regardless of success or failure. Therefore, I think we should wrap this test in a `describe` with an `after` function to contain the cleanup activities.